src/Filter/LabelFilter/LabelFilter.tsx

```diff
@@ -59,68 +56,41 @@ function LabelFilterPopper({
   );
 }
 
-/**
- * An inline label filter is always open and the popper does not sit above other elements.
- */
-function LabelFilterAlwaysOpen({
-  name,
-  label,
-  options,
-  value,
-  limitTags,
-  onChange
-}) {
-  return (
-    <AlwaysOpenAutocomplete
-      getOptionLabel={option => option.name}
-      isOptionEqualToValue={(option, value) => option._id === value._id}
-      limitTags={limitTags}
-      multiple
-      noOptionsText="No labels"
-      onChange={(e, newValue) => onChange(newValue)}
-      options={options}
-      renderInput={params => {
-        return (
-          <TextField
-            {...params}
-            label={label}
-            name={name}
-            sx={{
-              [`& .${autocompleteClasses.popupIndicator}`]: { display: "none" }
-            }}
-          />
-        );
-      }}
-      renderOption={Option}
-      renderTags={Tags}
-      value={value}
-    />
-  );
-}
-
 // render for label chips
-function Tags(labels, getTagProps, { onChange, value }) {
-  return labels.map(label => (
+function Tags(
+  labels: Label[],
+  getTagProps: AutocompleteRenderGetTagProps,
```

Comment:
Types should live seperality. `getTagProps` seems unsed?

Reply:
The getTagProps parameter is part of the expected signature for the renderTags function in Material-UI's Autocomplete component. Removing it would result in a type mismatch error.